src/rofi_rbw/rofi_rbw.py

```diff
@@ -171,15 +185,21 @@ def get_data(self, name: str, folder: str) -> Data:
             encoding='utf-8'
         ).stdout.split('\n')
 
-        password = result[0].strip()
-        username = self.extract_username(result)
+        password = result[0]
+        username = self.extract_data("Username", result)
+        totp = self.extract_data("TOTP Secret", result)
+
+        if "pyotp" in sys.modules and totp != "":
+            totp = pyotp.parse_uri(totp).now()
+        else
+            totp = ""
 
-        return Data(username, password)
+        return Data(username, password, totp)
```

Comment:
I'm wondering if `Credentials` is a better name than `Data`. Thoughts?

Reply:
I think `Data` is currently fine, since there is not much more data to reason about. However if there ever needs to be more "data" (I don't see why) then `Credentials` would be better. Doesn't hurt to change into `Credentials` atm anyway.